terraform/eval_diff.go

```diff
@@ -532,7 +531,7 @@ func processIgnoreChangesIndividual(prior, proposed cty.Value, ignoreChanges []h
 		// away any deeper values we already produced at that point.
 		var ignoreTraversal hcl.Traversal
 		for i, candidate := range ignoreChangesPath {
-			if reflect.DeepEqual(path, candidate) {
+			if path.Equals(candidate) {
```

Comment:
Should the comment above this be updated? There's all this detail `	// We're looking for an exact match here because this walk will visit` that might not be true since you're not doing `DeepEqual`

Reply:
Nope, the intended behavior is exactly the same here, it's just that `reflect.DeepEqual` doesn't compare the correct values, since `IndexStep`s can contain `big.Float`s.

Reply:
Thanks!